examples/app-mssql/main.py

```diff
@@ -2,24 +2,26 @@
 import sys
 import os
 import time
+from typing import List, Tuple
+
 import pyodbc
+from pyodbc import Cursor, Connection
```

Comment:
You've already imported the entire `pyodbc` module above. Why import these again?

Reply:
without specifying the imports, the program raises `NameError` exception.